src/Pages/sunkyungnoh/Main/components/Header.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Header.scss';
+
+class Header extends React.Component {
+  render() {
+    const { userId } = this.props;
+    return (
+      <header>
+        <div className="userPic header"></div>
+        <p className="userID">{userId}</p>
+        <button>
+          <div></div>
```

Comment:
```suggestion
          <div />
```
- 셀프 클로징 태그 이용해줄 수 있을 것 같습니다.
- 도형으로 사용되는 경우 i 태그를 사용하기도 합니다!

Reply:
폰트어썸에서 봤던 태그네요!! i 태그로 수정했습니다!